src/wasm-traversal.h

```diff
@@ -716,13 +718,17 @@ struct PostWalker : public Walker<SubType, VisitorType> {
   }
 };
 
+// Stacks of expressions tend to be limited in size (although, sometimes
+// super-nested blocks exist for br_table).
+typedef SmallVector<Expression*, 10> ExpressionStack;
```

Comment:
Why 10? I'd think 4 or 8 would be more cache-friendly, though I guess it depends on how big `std::vector` is

Reply:
10 pointers on 64-bit would be a multiple of 16, so seems like it should be ok cache-wise, or am I missing something?

I kind of picked 10 because I didn't see a benchmark advantage to smaller or larger values, and it seems reasonable as a guess for a "typical" wasm function body.